enaml/applib/live_editor_view.enaml

```diff
@@ -47,22 +49,40 @@ enamldef EditorPanel(Container):
         minimum = -10
         maximum = 20
         value := editor.zoom
+    Label: tlabel:
+        text = "Theme"
+    ObjectCombo: tcmb:
+        items = sorted(THEMES.keys())
+        selected = 'idle'
     Scintilla: editor:
         syntax = 'enaml'
-        theme = IDLE_THEME
+        theme << THEMES[tcmb.selected]
         settings = {
             "tab_width": 4,
             "use_tabs": False,
             "indent": 4,
             "tab_indents": True,
             "backspace_unindents": True,
+            "autocompletion_threshold": 1,
+            "show_line_numbers": True,
         }
+        autocomplete = 'all'
+        autocompletions = ['enamldef']
```

Comment:
I believe this should include all enaml keywords (attr, alias, enamldef, func, template, event). WDYT ?

Reply:
The `autocompletions` get's updated by the model so this is really only there so you can see how it works.

Reply:
Ok in this case what bothers me is that jedi will not propose enaml keyword for completion and in the live editor it would be nice to have them.

Reply:
Yes, I'd also like this. 

The keywords could simply be added [here](https://github.com/frmdstryr/enaml/blob/latest/enaml/applib/live_editor_model.py#L254) which will make it suggest them all the time. The problem is that that will make them show up everywhere, when the results should be context aware. It would be even more powerful if it could intelligently autocomplete declarative members when you're in a childef block.  

I did look though jedi's source to see how involved this would be and found that Jedi abstracts this part out into it's https://github.com/davidhalter/parso library.  Maybe could be somehow tied in with the enaml parser but that's as far as I got into it.

Reply:
I somehow though that Scintilla was responsible for filtering the list of possible auto completion it was provided with to make meaningful suggestions. Anyway I still need to take some time to test this myself before merging. Hopefully it will not be too long.